knowledge-base/objectdatasource-assembly-not-recognized-by-standalone-designer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ObjectDataSource Assembly Not Recognized by the Standalone Designer
+description: How to troubleshoot which assemblies to add to the Standalone designer folder when an ObjectDataSource is not property recognized
+type: troubleshooting
+page_title: 
+slug: objectdatasource-assembly-not-recognized-by-standalone-designer
+position: 
+tags: 
+ticketid: 1453347
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When you use an ObjectDataSourcem, which DataSource has external dependencies, the Standalone designer may not recognize the main ObjectDataSource assembly even when it is properly added to the designer folder and registered in its configuration file.
```

Comment:
The Standalone Designer may not recognize the main ObjectDataSource assembly (even though the assembly is properly added to the designer folder and registered in its configuration file) when this assembly depends on other assemblies.